sdk/monitor/Azure.Monitor.OpenTelemetry/tests/Azure.Monitor.OpenTelemetry.Demo/appsettings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "AzureMonitorOpenTelemetry": {
+        "ConnectionString": "InstrumentationKey=00000000-0000-0000-0000-000000000000",
+        "DisableOfflineStorage": false,
+        "EnableLogs": true,
```

Comment:
These are the default settings right? If the user omitted them the behavior would still be the same I think?

Reply:
Correct, except the `ConnectionString` all others honor default settings. If `ConnectionString` is not provided either through config or environment variable `APPLICATIONINSIGHTS_CONNECTION_STRING` an exception is thrown.